src/panel-manager.js

```diff
@@ -5,10 +5,22 @@
 /** @const */
 genotet.panelManager = {};
 
-/** @const {number} */
-genotet.panelManager.COLLAPSED_WIDTH = 32;
-/** @const {number} */
-genotet.panelManager.TRANSITION_TIME = 300;
+/** @private @const {number} */
+genotet.panelManager.COLLAPSED_WIDTH_ = 32;
```

Comment:
genotet.panelManager.COLLAPSED_WIDTH is used in viewManager (line 23) and needs to be public variable.